docs/src/main/asciidoc/hibernate-orm-panache.adoc

```diff
@@ -1,326 +1,752 @@
-        delete("name", "Stef");
-    }
-}
-----
+For an example of a completed {ProductName} Maven project that uses the Panache extension, see the https://github.com/quarkusio/quarkus-quickstarts/tree/master/hibernate-orm-panache-quickstart[Hibernate ORM Panache Quickstart].
 
-You have noticed how much more compact and readable the code is?
-Does this look interesting? Read on!
+== Hibernate ORM
 
-NOTE: the `list()` method might be surprising at first. It takes fragments of HQL (JP-QL) queries and contextualizes the rest. That makes for very concise but yet readable code.
+//adapted from https://docs.jboss.org/hibernate/orm/6.0/userguide/html_single/Hibernate_User_Guide.html. Module will be reused in Hibernate ORM story.
 
-== Setting up and configuring Hibernate ORM with Panache
+Coupling Java object-oriented software projects with relational databases for data persistence can be cumbersome and time-consuming. Development costs are significantly higher due to a paradigm mismatch between how data is represented in objects versus relational databases. Hibernate is an Object/Relational Mapping (ORM) solution for Java environments that facilitates mapping data to and from an object model representation to a relational data model representation. It is an implementation of the Java Persistence API (JPA).
 
-To get started:
+Hibernate not only takes care of the mapping from Java classes to database tables and from Java data types to SQL data types, but also provides data query and retrieval facilities. It can significantly reduce development time otherwise spent with manual data handling in SQL and Java database connectivity (JDBC). Hibernate’s design goal is to relieve you of most common data persistence-related programming tasks by eliminating the need for manual data processing using SQL and JDBC.
 
-* add your settings in `{config-file}`
-* annotate your entities with `@Entity` and make them extend `PanacheEntity`
-* place your entity logic in static methods in your entities
+The JPA `@entity` annotation is at the heart of Hibernate ORM. When applied to a Java class, it maps the class to a specific database table. The fields in the class map to database columns. Each instance of the class represents a database row.
 
-Follow the link:hibernate-orm#setting-up-and-configuring-hibernate-orm[Hibernate set-up guide for all configuration].
+== Hibernate ORM with the {ProductName} Panache JPA extension
 
-In your `pom.xml`, add the following dependencies:
+Hibernate ORM is the most widely used JPA implementation. It provides all of the features of an ORM. However, for a simple implementation with the most common data mappings you can use the {ProductName} Panache JPA extension to write more concise code.
 
-* the Panache JPA extension
-* your JDBC driver extension (`quarkus-jdbc-postgresql`, `quarkus-jdbc-h2`, `quarkus-jdbc-mariadb`, ...)
+The Hibernate ORM with Panache extension contains the following classes:
 
-[source,xml]
+* `PanacheEntity`
```

Comment:
I feel like this list of types is not very useful to people reading the guide, and might scare people away from keeping on reading, especially compared to the previous intro which went straight to the point with a short example that would instantly hook any existing ORM user and entice them into reading.

Reply:
I'll leave this to Nicole and Vidya to comment.

Reply:
I'm okay removing them if it's not helpful to the customer and will only cause confusion.

Reply:
Yeah, it's not useful.

Reply:
So it's not useful to tell the user  what classes the extension contains? It's a short list and the reason I included it is so that when we refer to PanacheEntity and PanacheEntityBase there is context for the reader.